contribute/developer-guide.md

```diff
@@ -25,6 +25,10 @@ brew install node@14
 npm install -g yarn
 ```
 
+### windows
```

Comment:
```suggestion
### Windows
```

Reply:
Done! 🤗

Reply:
Thanks, @oddlittlebird for the corrections!